Source/WebKit/NetworkProcess/mac/NetworkConnectionToWebProcessMac.mm

```diff
@@ -60,6 +68,53 @@
 #endif
 }
 
-#endif // PLATFORM(MAC)
+#if ENABLE(LAUNCHSERVICES_SANDBOX_EXTENSION_BLOCKING)
+void NetworkConnectionToWebProcess::checkInWebProcess(CoreIPCAuditToken&& auditToken)
+{
+    RELEASE_LOG(Process, "NetworkConnectionToWebProcess::checkInWebProcess");
+
+    int dyldPlatform = dyld_get_active_platform();
+    auto dyldPlatformValue = adoptCF(CFNumberCreate(kCFAllocatorDefault, kCFNumberIntType, &dyldPlatform));
```

Comment:
Please do not hide `RetainPtr` behind `auto` in new code. The same comment applies elsewhere in this PR.